module/zfs/spa_misc.c

```diff
@@ -244,7 +244,7 @@ int spa_mode_global;
 
 #ifdef ZFS_DEBUG
 /* Everything except dprintf and spa is on by default in debug builds */
```

Comment:
nit: Should this comment be updated?

Reply:
I'd be more inclined to remove the comment.  It's just restating the code in English.